autofaiss/indices/distributed.py

```diff
@@ -304,12 +326,13 @@ def run(
                 start=x[1],
                 end=x[2],
                 memory_available_for_adding=memory_available_for_adding,
-                broadcast_trained_index_bytes=broadcast_trained_index_bytes,
+                broadcasted_index_or_path=broadcasted_index_or_path,
```

Comment:
The trained keyword is relevant, it is a different state of an index that one can check with something like `index.is_trained()`. I would have kept here (you can't `index.add()` on a not-trained index.